src/components/map/MapDirective.js

```diff
@@ -79,6 +88,49 @@ goog.require('ga_permalink');
 
             map.setTarget(element[0]);
 
+            scope.$watch('::ol3d', function(ol3d) {
```

Comment:
I think so: the two expressions uses one time binding:
 - here with `scope.$watch('::ol3d', function(ol3d) {`
 - in `index.mako.html` with `<div ga-map ga-map-map="map" ga-map-ol3d="::ol3d">`